app/assets/javascripts/hyrax/app.js

```diff
@@ -134,5 +135,12 @@ Hyrax = {
     statistics: {},
 
     // initialized in hyrax/config.js
-    config: {}
+    config: {},
+
+    // Per Page select will submit its form to change records shown
+    batchSubmit: function () {
+        var BatchSubmit = require('hyrax/batch_select');
```

Comment:
@jcoyne This matches the other requires in the system.  Any idea why hound is yelling for this one? Should it just be ignored?